modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1177 @@
+
+As a consequence, a bug was identified that by using the `execCommands`, which are provided by {productname}, the user could modify the table element regardless of it being nested within a `contenteditable="false"` root element. This was made possible by by overriding the `contenteditable="false"` element settings by using the `execCommand`'s' to trigger the `action` event.
+
+To fix this bug, {productname} 6.4, made improvements to the DOM `Commands` API that now blocks all Table UI actions via the `execCommands` when a `contenteditable="false"` element selection is highlighted inside a `contenteditable="false"` root.
+
+As a result, `contenteditable="false"` elements such as tables cannot have their properties overridden by using the `execCommands`.
+
+_(Readmore about editorCommands xref:editor-command-identifiers.adoc[here])_
+
+=== Fake carets were rendered for noneditable elements and tables in a noneditable root element
+// TINY-9459
+
+In previous versions of {productname}, `fake` carets that are within a `contenteditable="false"` element, would render as a `highlighted` node, when nested within another `contenteditable="false"` element, this would occured when the content was placed within a `table` placed in the root element.
+
+As a consequence, `fake` caret/s would be visibly highlighted within the editor viewport,  which would allow content to be inserted into the element when the table was selected by using either the keyboard and mouse function commands.
+
+To fix this {productname} 6.4, added a new function to the Utils API that filters any content that is considered as a `fake caret` element, which now disables the users ability to add/insert or modify the content contained within the `contenteditable="false"` element node space.
+
+As a result, any `contenteditable="false"` elements that are nested within another `contenteditable="false"` element root that are nested within `table` elements, can no longer be edited in {productname} 6.4.
+
+=== Textareas with scrollbars in dialogs would not render rounded corners correctly on some browsers
+// TINY-9331
+
+In previous versions of {productname}, the source code visible UI boarder would not render correctly in some browsers.
+
+As a consequence, when viewing text-areas that contain `scrollbars` within dialogs, the `focus` visible boarder was not rendering the corners inside the dialog.
+
+To correct this issue, {productname} 6.4, introduced a new text-area wrapper that fixes the issues with the `boarder-color, radius, style and width` that directly affects the `focus` render element for the dialog.  In addition `overflow: hidden;`, has been added to clip the scrollbars to the border-radius.
+
+As a result, the `focus` dialog boarder now renders correctly in {productname} 6.4.
+
+=== It was possible to open links inside the editor if the editor root was noneditable
```

Comment:
Suggest to look the ticket.

Reply:
@tiny-ben-tran note, this has been fixed and the new changes have been applied.